shopelectro/tests/tests_views.py

```diff
@@ -527,20 +531,33 @@ def test_tags_pagination_has_canonical_links(self):
             )
         )
 
-    def test_category_matrix_page(self):
-        """Matrix page should contain all second level categories."""
-        page = CustomPage.objects.get(slug='catalog')
-        response = self.client.get(page.url)
-        soup = BeautifulSoup(
-            response.content.decode('utf-8'),
-            'html.parser'
-        )
-        self.assertEqual(200, response.status_code)
-
-        categories_db = models.Category.objects.active()
-        categories_app = soup.find_all(tag='a', class_='catalog-list-item')
-        for from_db, from_app in zip(categories_db, categories_app):
-            self.assertEqual(from_db.name, from_app.a.text)
+    # @todo #887:30m  Fix not active crumbs siblings bug.
+    #  The test below proves the bug. Now fix it.
+    @unittest.expectedFailure
+    def test_crumb_siblings_are_active(self):
+        parent = models.Category.objects.raw(
```

Comment:
It seems it is the same, but simpler 
```suggestion
        parent = models.Category.objects.annotate(c=Count('children')).filter(c__gt=1).first()
```

Reply:
thanks! I'll try it